### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_BlazorServerManageurAccntPWD_NonTextContrast: The Color contrast ratio of the focus indicator of the "Update password" button is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42373]
![42373_A11y_ NET Core_ASP NETCore_BlazorServerManageurAccntPWD_NonTextContrast](https://user-images.githubusercontent.com/86710491/175509798-54212d2e-74cd-4e37-9620-80748a90df20.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir BlazorServer"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new blazorserver -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. To get access to all the pages you'll need to register first and then log-in.
10. Login with your registered account 
11. Tab till "Hello with the userid" and click on it.
12. Navigate to Password and click on it
13. Turn on Color contrast analyzer and check that The Color contrast ratio of "Update password" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Update password" button is less than 3:1.

**This issue is observed throughout the application for all the "Update password" buttons.**

### Expected:
The Color contrast ratio of the focus indicator of the  "Update password" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/aspnetcore/pull/43177

**Comment by merriemcgaw** [comment]
@Truptik12 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached.

![image](https://user-images.githubusercontent.com/86710491/192523693-da088224-d71d-4777-bd67-43846192558a.png)
**

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
